conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+# FIX Connect
+
+This "Connect" component extension supports connection to the target system via [FIX protocol](https://www.fixtrading.org/what-is-fix/).
+
+## Available service types
+
+This "Connect" component provides users with two types of the services:
+
+### FIX Client
+
+The "Connect" component acts like a client that connects to the system.
```

Comment:
The "Connect" component acts like a client that connects to the **target** system.